### DISCUSSION TITLE
Profile for Deutsche Telekom

### DISCUSSION THREAD

**Discussion by MirkoKas** [fabianishere/udm-iptv#128] [Show and tell]
As I setup my UDM Pro a couple of days ago and had a bit of research to do, to find the right configuration setup for Deutsche Telekom and their product MagentaTV, I created a new profile for easier use for other German users.
I used the KPN one as a sample and hope everything is done correctly.

@fabianishere : You are free to add this to the repository for future releases. 

```
#!/bin/sh -e
# Profile for Deutsche Telekom (DE)
#
# Copyright (C) 2022 Mirko Kasparek.
#
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version.

db_get udm-iptv/wan-port
db_set udm-iptv/wan-interface "$RET"

db_set udm-iptv/wan-vlan 7
# only set the WAN-VLAN if you haven’t enabled it on your modem already
db_set udm-iptv/wan-ranges "232.0.0.0/16, 87.141.0.0/16"
db_set udm-iptv/wan-dhcp-options "-O staticroutes -V IPTV_RG"
```

**Comment by MirkoKas** [comment]
Not able to get the layout of my post right on iOS- sorry for that

**Comment by fabianishere** [comment]
Any firewall rules to take into account?

**Reply by MirkoKas** [reply]
Yes, you have to set the rules posted here: https://www.bjoerns-techblog.de/2017/06/deutsche-telekom-entertain-mit-dem-ubiquiti-unifi-security-gateway/

I could add the rules as a comment within the profile if this would make sense. Like I did with setting the WAN-VLAN already.
